src/snowflake/snowpark/modin/plugin/extensions/pd_extensions.py

```diff
@@ -23,11 +23,14 @@
 from snowflake.snowpark.modin.plugin.utils.warning_message import (
     materialization_warning,
 )
+from snowflake.snowpark.modin.plugin.extensions.configs import Configs
 
 register_pd_accessor("Index")(Index)
 register_pd_accessor("DatetimeIndex")(DatetimeIndex)
 register_pd_accessor("TimedeltaIndex")(TimedeltaIndex)
 
+register_pd_accessor("configs")(Configs)
```

Comment:
I'm unsure about adding a distinct `pd.configs` variable for settings like this. modin and native pandas already provide their own configuration mechanisms:
- [`modin.config`](https://modin.readthedocs.io/en/stable/flow/modin/config.html) variables control partitioning and other behavior in OSS modin; we currently still use a vendored copy of `modin/config.py`
- [`pd.set_option`/`pd.get_option`](https://pandas.pydata.org/docs/user_guide/options.html) controls display options and some other configuration for native pandas, which we sometimes use in Snowpark pandas

It's possible for us to add a Modin environment variable since we still have a vendored copy, but this might not be a solution that works in the long term. I don't know if it's possible for us to add new items to `pd.set_options`.

I would prefer renaming this new variable to something like `pd.snowflake_options` to make clear that it is specific to Snowpark pandas, and avoid confusing users into thinking it's a native pandas API.

Reply:
In fact, i can not do it like this. This will not work for native app. i need to do it in the snowflake way. I need to close this pr